### PR TITLE
Language and typos in documentation/code.md

### PR DESCRIPTION
**Pull request by gkaracha**
Just fixes for a couple of minor issues I came across while reading the docs :-)

### CODE REVIEW

File: documentation/code.md [hunk 1]
```diff
@@ -81,8 +82,7 @@ files explaining a package. This hierarchy must include pointers to:
           when there are other possible choices.
 - the main entry points in the code for understanding the
   implementation,
-- resources for learning the libraries and frameworks on which the
-  package depends.
+- resources for learning the libraries and frameworks the package depends on.
```
Comment (aspiwack) [inline documentation/code.md:85]: Actually, the original is better. Let's revert this one.

File: documentation/code.md [hunk 2]
```diff
@@ -222,7 +222,7 @@ follow them, they could be incorporated together with new changes.
   is, the hierarchy of text files needs to be fleshed out sufficiently
   so the new state can be communicated to a person unfamiliar with the
   package. Examples:
-    - The scope of the solution a package implements changes as a
+    - The scope of the solution a package implements changes, as a
```
Comment (aspiwack) [inline documentation/code.md:225]: I'm pretty sure this is wrong.

Comment (gkaracha) [reply documentation/code.md:225]: Ha, the only reason I added the comma was for consistency; the next item has it too. I will instead remove it from both :)

File: documentation/code.md (hunk 1 above)
Comment (gkaracha) [reply documentation/code.md]: Done.

File: documentation/code.md (hunk 2 above)
Comment (gkaracha) [reply documentation/code.md]: Done.

**Review by facundominguez — APPROVED** [review]
Thanks @gkaracha!

**Pull request merged**